c/doc/nnstreamer_doc.h

```diff
@@ -97,5 +97,27 @@
  * </a>
  */
 
+/**
+ * @ingroup  CAPI_ML_FRAMEWORK
+ * @defgroup CAPI_ML_NNSTREAMER_SERVICE_MODULE Service
+ * @addtogroup CAPI_ML_NNSTREAMER_SERVICE_MODULE
+ * @brief The NNStreamer Service API provides interfaces to store and fetch the pipeline description for AI application developers.
+ * @section CAPI_ML_NNSTREAMER_SERVICE_HEADER Required Header
+ *   \#include <nnstreamer/ml-api-service.h>\n
+ *
+ * @section CAPI_ML_NNSTREAMER_SERVICE_OVERVIEW Overview
+ * The NNStreamer Service API provides utility interfaces for AI application developers.
+ *
+ * This function allows the following operations with NNStreamer:
+ * - Set and get the pipeline description with a given name.
+ * - Deletes the pipeline description with a given name.
```

Comment:
Deletes > Delete

Reply:
For future work? We have plan to add more Service APIs - nn model, offload, ...
`Manage the pipeline description for AI application developers.`